collector/os_release.go

```diff
@@ -195,6 +210,9 @@ func (c *osReleaseCollector) Update(ch chan<- prometheus.Metric) error {
 		ch <- prometheus.MustNewConstMetric(c.versionDesc, prometheus.GaugeValue, c.version,
 			c.os.ID, c.os.IDLike, c.os.Name)
 	}
+
+	ch <- prometheus.MustNewConstMetric(c.supportEndDesc, prometheus.GaugeValue, float64(c.supportEnd.Unix()))
```

Comment:
Do we want to expose this always? Or skip the metric if there is no `SUPPORT_END`.

Reply:
I'm going to just skip if the distro doesn't ship this optional extra.